src/js/tools/metroconfig.ts

```diff
@@ -146,6 +157,26 @@ function withSentryDebugId(config: MetroConfig): MetroConfig {
   };
 }
 
+function excludeSentryWebReplay(config: MetroConfig): MetroConfig {
+  const originalResolver = config.resolver?.resolveRequest;
+
+  return {
+    ...config,
+    resolver: {
+      ...config.resolver,
+      resolveRequest: (context, moduleName, platform) => {
```

Comment:
Would it make sense to include web replay on `platform=web`?

So it work for web out of the box and when users want to optimalize the bundle size and are not using Replay they can use the option `includeWebReplay`.